src/test/java/io/github/kawamuray/wasmtime/ConfigTest.java

```diff
@@ -0,0 +1,47 @@
+package io.github.kawamuray.wasmtime;
+
+import org.junit.Test;
+
+public class ConfigTest {
+    @Test
+    public void testNewConfig(){
+        Config config = new Config();
+        assert config.innerPtr()!=0;
+    }
+    @Test
+    public void testCacheConfigLoadDefault(){
+        Config config = new Config();
+        config.cacheConfigLoadDefault();
+    }
+    @Test
+    public void testStrategy(){
+        Config config = new Config();
+        config.strategy(Strategy.AUTO).strategy(Strategy.CRANELIFT);
+    }
+    @Test
+    public void testOptLevel(){
+        Config config = new Config();
+        config.strategy(Strategy.CRANELIFT).craneliftOptLevel(OptLevel.SPEED_AND_SIZE);
+        Config config1 = new Config();
+        config1.strategy(Strategy.CRANELIFT).craneliftOptLevel(OptLevel.SPEED);
+        Config config2 = new Config();
+        config2.strategy(Strategy.CRANELIFT).craneliftOptLevel(OptLevel.NONE);
+        Config config3 = new Config();
+        config3.strategy(Strategy.AUTO).craneliftOptLevel(OptLevel.SPEED_AND_SIZE);
+        Config config4 = new Config();
+        config4.strategy(Strategy.AUTO).craneliftOptLevel(OptLevel.SPEED);
+        Config config5 = new Config();
+        config5.strategy(Strategy.AUTO).craneliftOptLevel(OptLevel.NONE);
+    }
+
+    @Test
+    public void testCustomConfig(){
+        Config config = new Config();
+        config.strategy(Strategy.CRANELIFT)
+                .craneliftOptLevel(OptLevel.SPEED_AND_SIZE)
+                .debugInfo(true);
+        Engine engine = new Engine(config);
+        Store store = new Store(engine);
+        store.engine();
```

Comment:
@kawamuray `assertEquals(engine, store.engine());` will throw an Expection all the time because method engine() builds a new Engine class. Is there any other ways to do this?

Reply:
Ah, my bad. `Store::new` takes only a reference to `Engine` and [uses `clone()` to shallow copy an Rc reference](https://github.com/bytecodealliance/wasmtime/blob/main/crates/wasmtime/src/store.rs#L103) so their pointer address which is pointing outer object doesn't match. 
Then I think it's difficult to test two object's equality without getting help from another JNI methods. So I think we can simply get rid of L43-L45 which is not the main subject of this test case.